### DISCUSSION TITLE
users interview

### DISCUSSION THREAD

**Discussion by johnedelbi** [MichaelMugaiga/MichaelMugaiga.github.io#8] [Show and tell]
hello everyone would you give me your comment about the book store for 

you can see the prototype here 
https://github.com/MichaelMugaiga/MichaelMugaiga.github.io/blob/main/planning/05-paper-prototype.md

and here you can find our interview 
https://github.com/MichaelMugaiga/MichaelMugaiga.github.io/blob/main/planning/01-stakeholder-interviews.md

**Comment by V-Valkiriya** [comment]
Hello, John. I think your prototype is very nice. It`s clear and full. 

The only thing I haven't found is where the used books will be. It seems to me that some eco-activists or people who want to save money would prefer to choose from them at once. But perhaps you have provided that they will be shown among genres.

**Reply by johnedelbi** [reply]
yes for sure i think i forget that part :) i will add it thx

**Comment by inna9Z** [comment]
@johnedelbi very interesting prototype, I like it, well done )

**Comment by Izoku** [comment]
Hi John. Your prototype is very clear, I also like that in the interview you asked who where the main competitors of your stakeholder. 

For the slider/carrousel you have on top, you may want to consider some kind of element that tells you how many elements or pictures are in the slider .

![image](https://user-images.githubusercontent.com/7308549/187075688-78bd0a1b-5e31-4486-96b6-0e73bf8e23fe.png)

**Reply by johnedelbi** [reply]
ok thx i will update my design
